src/faldbt/lib.py

```diff
@@ -338,6 +338,9 @@ def _write_relation(
     *,
     dtype=None,
 ) -> AdapterResponse:
+    if adapter.type() == "fal":
+        adapter = adapter._db_adapter
+
```

Comment:
This addresses FEA-594

Reply:
Awsome! Not necessary for fetch relations too?

Reply:
Yep!